### DISCUSSION TITLE
how to stup mgt commerce on different port than 80

### DISCUSSION THREAD

**Discussion by alam-ejaz** [mgtcommerce/mgt-dev#19] [General]
one od my applications is already using port 80. I tried to setup MGT Commerce on Port 86 but it didnt work. Is there any other way?>

**Comment by wmdkdkussin** [comment]
@alam-ejaz Do you want port 80 to be used by another application or don't you know which application it is?

**Reply by alam-ejaz** [reply]
actually when I run **net stop http** on my windows laptop then the port 80 for mgt commerce starts working. after the laptop restarts, then port 80 get occupied by http service.
i want to use a different port for mgt commerce.
